Homeworks/module13.py

```diff
@@ -0,0 +1,39 @@
+import time
```

Comment:
Why? I didn't notice any usage

Reply:
I think I just used PyCharm and overwritten an exiting .py file at the same time forgetting to erase first line